File: CCC-2019/S2.cpp
#include <bits/stdc++.h>

using namespace std;

bool isPrime(int num)
{
    for (int i=2;i<=sqrt(num);i++)
    {
        if (num%i == 0) {return false;};
    };
    return true;
};

int main()
{
    ios_base::sync_with_stdio(0); cin.tie(0); cout.tie(0);
    int a,b; cin >> a;
    vector<int> nums;
    for (int i=0;i<a;i++)
    {
        cin >> b; nums.push_back(b);
    };
    for (int i=0;i<a;i++)
    {
        int n = nums[i];
        for (int j=3;j<=n-1;j++)
        {
            if (isPrime(j) && isPrime((n*2)-j))
            {
                cout << j << ' ' << (n*2)-j << endl; 
                break;
            };
        };
    };
};

File: CCC-2019/Python/S2.py
def isPrime(num):
    for i in range(2, int(num**0.5)+1):
        if num%i == 0:
            return False 
    return True 

t = int(input())

for i in range(0, t):
    n = int(input())
    for j in range(3, n+1):
        if isPrime(j) and isPrime(n*2-j):
            print(f'{j} {n*2-j}')
            break

